src/components/study-container.jsx

```diff
@@ -718,7 +718,7 @@ export function StudyContainer({ view, onChangeTab }) {
         }
         let previousCurrentNode = currentNodeRef.current;
         currentNodeRef.current = currentNode;
-
+        let previousCurrentRootNetwork = currentRootNetworkRef.current;
```

Comment:
```suggestion
        const previousCurrentRootNetwork = currentRootNetworkRef.current;
```

Reply:
true, will fix it in the following refactor PR because it's also true for `previousCurrentNode` variable here